content/project/conduct.adoc

```diff
@@ -98,27 +100,30 @@ Those hosted on Google Groups (see
 link:/mailing-lists[here]) and those self-hosted
 on link:http://lists.jenkins-ci.org/mailman/listinfo[lists.jenkins-ci.org].
 
-==== IRC Channels
+==== Chats
 
-Documented in the
-link:/chat[IRC Channel] wiki
-page, e.g.
+All chats documented on the link:/chat[this page], e.g.
 
-* `#jenkins`
-* `#jenkins-meeting`
-* `#jenkins-infra`
-* `#jenkins-community`
+* IRC channels: `#jenkins`, `#jenkins-meeting`, `#jenkins-infra`, etc.
+* Gitter channels within the link:https://gitter.im/jenkinsci/home[https://gitter.im/jenkinsci/] space
 
 ==== Events
 
-* link:/projects/jam/[Jenkins
-  Area Meetup]
+* link:/projects/jam/[Jenkins Area Meetups]
+* link:/events/online-meetup[Jenkins Online Meetup]
 * Events where Jenkins is officially being represented (e.g.
   link:https://fosdem.org[FOSDEM], link:https://socallinuxexpo.org/[SCaLE])
 * Other Jenkins groups (such as conferences, meetups, and other unofficial
   forums) are encouraged to adopt this Code of Conduct. Those groups must
   provide their own moderators and/or reporting system.
 
+==== Social media
+
+Official Jenkins community accounts and channels, including:
+
+* Twitter: link:https://twitter.com/jenkinsci[@jenkinsci], link:https://twitter.com/jenkins_release[@jenkins_release]
+* LinkedIn: link:https://www.linkedin.com/company/jenkins-project/[jenkins-project]
+* other social media accounts
```

Comment:
IMHO this list must be more deterministic, so this last entry should be modified to say "Other social media accounts managed by the Jenkins community" or something like that

Reply:
```suggestion
* Other social media accounts managed by the Jenkins community
```